### ISSUE TITLE
Not applying

### ISSUE THREAD

**Issue by disc0nct** [sandesh236/sleek--themes#6]
sorry to hear that but install.sh only changes grub configuration, it doesn't make any other changes.
my distro is kali linux.
![image](https://user-images.githubusercontent.com/17936240/135473637-55ca32c5-22b1-4392-9a88-c15658cbb742.png)

**Comment by sandesh236** [comment]
could you please share the content of "/etc/default/grub" file...

**Comment by dhrp01** [comment]
I had the same issue.
I was executing it as `sudo ./Sleek\ theme-bigSur/install.sh`. Although it worked, it didn't set the theme appropriately.
What worked for me was `cd Sleek\ theme-bigSur && ./install.sh`.
Minor issue, but maybe worth mentioning, if you are doing the same.

**Comment by devangshekhawat** [comment]
I had the same issue of Fedora 36.
Shows that it worked but actually nothing changes.

**Comment by devangshekhawat** [comment]
> could you please share the content of "/etc/default/grub" file...

```
GRUB_TIMEOUT=5
GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/system-release)"
GRUB_DEFAULT=saved
GRUB_DISABLE_SUBMENU=true
GRUB_TERMINAL_OUTPUT="console"
GRUB_CMDLINE_LINUX="rd.driver.blacklist=nouveau modprobe.blacklist=nouveau nvidia-drm.modeset=1 initcall_blacklist=simpledrm_platform_driver_init rhgb quiet rd.driver.blacklist=nouveau modprobe.blacklist=nouveau nvidia-drm.modeset=1 initcall_blacklist=simpledrm_platform_driver_init"
GRUB_DISABLE_RECOVERY="true"
GRUB_ENABLE_BLSCFG=true
GRUB_THEME="/usr/share/grub/themes/sleek/theme.txt"
```

It does seem to be pointing to the theme.

**Comment by sandesh236** [comment]
Could you please check if the sleek theme is successfully copied to  /usr/share/grub/themes folder. If the theme is not there, please consider copying it manually. Note: copy the sleek folder which is inside the main theme[Sleek theme bigsur] folder consisting install and uninstall script.

**Comment by devangshekhawat** [comment]
Yes, I just checked the theme folder is there along with all the files.
I deleted the it and copied the raw folder there but it still fetched no results.

**Comment by devangshekhawat** [comment]
@njakes nope, it didn't work.
I found a reddit post later saying that grub theming is unsupported on fedora.

**Comment by CentillionJinx** [comment]
same with debian...

**Comment by st1pps** [comment]
@devangshekhawat 
I was able to fix this exact problem today.

I replaced/added these lines in `/etc/default/grub`:
```
GRUB_TERMINAL_OUTPUT="gfxterm"
GRUB_GFXMODE="1920x1200x32,auto"
GRUB_GFXPAYLOAD_LINUX="keep"
```
Adjust the resolution as needed. Theoretically, the last two added lines are optional - with `GRUB_TERMINAL_OUTPUT="console"` you force text mode. 

Afterwards generate grub files:
```
sudo grub2-mkconfig -o /boot/grub2/grub.cfg
```
